pkg/driverbuilder/builder/amazonlinux.go

```diff
@@ -275,12 +353,7 @@ func fetchAmazonLinuxPackagesURLs(kv kernelrelease.KernelRelease, targetType Typ
 			if err != nil {
 				log.Fatal(err)
 			}
-			base := repo
-			if targetType == TargetTypeAmazonLinux2 {
-				base = amazonlinux2baseURL
-			}
-			href = strings.ReplaceAll(href, "../", "")
```

Comment:
This was useless now that `getResolvingURLs` supports url references: https://github.com/falcosecurity/driverkit/blob/master/pkg/driverbuilder/builder/builders.go#L55